src/AppBundle/Resources/views/restaurant/stats.html.twig

```diff
@@ -10,7 +10,7 @@
 
 <div class="row">
   <div class="col-xs-6">
-    <h3 class="nomargin">Du {{ start|localizeddate('medium', 'none') }} au {{ end|localizeddate('medium', 'none') }}</h3>
+    <h3 class="nomargin">{% trans %} basics.from {% endtrans %} {{ start|localizeddate('medium', 'none') }} {% trans %} basics.to {% endtrans %}  {{ end|localizeddate('medium', 'none') }}</h3>
```

Comment:
Instead of concatenating strings, you should use parameters: 

```
{% trans with { 
  '%start%': '...',
  '%end%': '...',
} %}basics.date_range{% endtrans %}
```

```yaml
basics.date_range: From %start% to %end%
```